<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewScheduleDialog</class>
 <widget class="QMainWindow" name="NewScheduleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>619</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>619</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>619</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>新建日程</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip {
	border: 2px solid #ffebcd;
	padding: 5px;
	border-radius: 5px;
	opacity: 200;
}
QPushButton {
	border: 2px solid #8f8f91;
	border-radius: 6px;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f6f7fa, stop: 1 #dadbde);
	min-width: 80px;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}
QPushButton:flat {
	border: none;
}
QPushButton:default {
	background-color: darkgray;
}
QLineEdit {
	border: 2px solid gray;
	border-radius: 10px;
	padding: 0 8px;
}
QPlainTextEdit {
	background-color: white;
	border: 2px solid gray;
	border-radius: 10px;
	padding: 2px;
}
QTextEdit {
	background-color: white;
	border: 2px solid gray;
	border-radius: 10px;
	padding: 2px;
}
QGroupBox {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e0e0e0, stop: 1 #ffffff);
	border: 2px solid gray;
	border-radius: 5px;
	margin-top: 1ex;
}
QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left;
	padding: 0 3px;
}
QFontComboBox {
	border: 1px solid gray;
	border-radius: 3px;
	padding: 1px 18px 1px 3px;
	min-width: 6em;
}
QFontComboBox:editable {
	background: white;
}
QFontComboBox:!editable, QFontComboBox::drop-down:editable {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e1e1e1, stop: 0.4 #dddddd, stop: 0.5 #d8d8d8, stop: 1.0 #d3d3d3);
}
QFontComboBox:!editable:on, QFontComboBox::drop-down:editable:on {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #d3d3d3, stop: 0.4 #d8d8d8, stop: 0.5 #dddddd, stop: 1.0 #e1e1e1);
}
QFontComboBox:on {
	padding-top: 3px;
	padding-left: 4px;
}
QFontComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 15px;
	
	border-left-width: 1px;
	border-left-color: darkgray;
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}
QFontComboBox::down-arrow {
	image: url(:/img/images/down-arrow.png);
}
QFontComboBox::down-arrow:on {
	top: 1px;
	left: 1px;
}
QFontComboBox QAbstractItemView {
	border: 2px solid darkgray;
}
QCheckBox {
	spacing: 5px;
}
QCheckBox::indicator {
	width: 13px;
	height: 13px;
}
QCheckBox::indicator:unchecked {
	image: url(:/img/images/checkbox_unchecked.png);
}
QCheckBox::indicator:unchecked:hover {
	image: url(:/img/images/checkbox_unchecked_hover.png);
}
QCheckBox::indicator:unchecked:pressed {
	image: url(:/img/images/checkbox_unchecked_pressed.png);
}
QCheckBox::indicator:checked {
	image: url(:/img/images/checkbox_checked.png);
}
QCheckBox::indicator:checked:hover {
	image: url(:/img/images/checkbox_checked_hover.png);
}
QCheckBox::indicator:checked:pressed {
	image: url(:/img/images/checkbox_checked_pressed.png);
}
QSpinBox {
	padding-right: 15px;
	border-image: url(:/img/images/frame.png) 4;
	border-width: 3;
}
QSpinBox::up-button {
	subcontrol-origin: border;
	subcontrol-position: top right;
	width: 16px;
	border-image: url(:/img/images/spinup.png) 1;
	border-width: 1px;
}
QSpinBox::up-button:hover {
	border-image: url(:/img/images/spinup_hover.png) 1;
}
QSpinBox::up-button:pressed {
	border-image: url(:/img/images/spinup_pressed.png) 1;
}
QSpinBox::up-arrow {
	image: url(:/img/images/up_arrow.png);
	width: 7px;
	height: 7px;
}
QSpinBox::up-arrow:disabled, QSpinBox::up-arrow:off {
	image: url(:/img/images/up_arrow_disabled.png);
}
QSpinBox::down-button {
	subcontrol-origin: border;
	subcontrol-position: bottom right;
	width: 16px;
	border-image: url(:/img/images/spindown.png) 1;
	border-width: 1px;
	border-top-width: 0;
}
QSpinBox::down-button:hover {
	border-image: url(:/img/images/spindown_hover.png) 1;
}
QSpinBox::down-button:pressed {
	border-image: url(:/img/images/spindown_pressed.png) 1;
}
QSpinBox::down-arrow {
	image: url(:/img/images/down_arrow.png);
	width: 7px;
	height: 7px;
}
QSpinBox::down-arrow:disabled, QSpinBox::down-arrow:off {
	image: url(:/img/images/down_arrow_disabled.png);
}
QScrollBar:vertical {
	max-width: 8px;
	border: none;
	margin: 0px, 0px, 0px, 0px;
	border-radius: 2px;
	background-color: lightgray;
}
QScrollBar:vertical:hover {
	margin: 0px, 0px, 0px, 0px;
	background-color: lightgray;
}
QScrollBar::handle:vertical {
	max-width: 8px;
	background-color: gray;
	border-radius: 2px;
}
QScrollBar::handle:vertical:hover {
	background-color: darkgray;
}
QScrollBar::add-line:vertical {
	height: 0px;
	background-color: transparent;
	subcontrol-position: bottom;
}
QScrollBar::sub-line:vertical {
	height: 0px;
	background-color: transparent;
	subcontrol-position: top;
}
QScrollBar::add-page:vertical {
	background-color: lightgray;
}
QScrollBar::sub-page:vertical {
	background-color: lightgray;
}
QDateTimeEdit {
	border: 1px solid gray;
	border-radius: 3px;
	padding: 1px 18px 1px 3px;
	min-width: 6em;
}
QDateTimeEdit:editable {
	background: white;
}
QDateTimeEdit:!editable {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e1e1e1, stop: 0.4 #dddddd, stop: 0.5 #d8d8d8, stop: 1 #d3d3d3);
}
QDateTimeEdit::drop-down:editable {
	background: transparent;
}
QDateTimeEdit:!editable:on {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #d3d3d3, stop: 0.4 #d8d8d8, stop: 0.5 #dddddd, stop: 0 #e1e1e1);
}
QDateTimeEdit::drop-down:editable:on {
	background: transparent;
}
QDateTimeEdit:on {
	padding-top: 3px;
	padding-left: 4px;
}
QDateTimeEdit::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 15px;
	
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}
QDateTimeEdit::down-arrow {
	image: url(:/img/images/Calendar.png);
}
QDateTimeEdit::down-arrow:on {
	top: 1px;
	left: 1px;
}
QPushButton#CalendarLeftYearBtn,
QPushButton#CalendarRightYearBtn,
QPushButton#CalendarLeftMonthBtn,
QPushButton#CalendarRightMonthBtn {
	border: none;
	border-radius: 0;
	background-color: transparent;
	width: 16px;
	height: 16px;
}
QPushButton#CalendarEnsureBtn,
QPushButton#CalendarTodayBtn {
	background-color: rgb(0, 161, 255);
	color: white;
}
CustomCalendarWidget {
	background-color: white;
}
QPushButton#CalendarLeftYearBtn {
	image: url(:/img/images/previous_year.png);
}
QPushButton#CalendarLeftYearBtn:hover,
QPushButton#CalendarLeftYearBtn:pressed {
	image: url(:/img/images/previous_year_hover.png);
}
QPushButton#CalendarLeftMonthBtn {
	image: url(:/img/images/previous_month.png);
}
QPushButton#CalendarLeftMonthBtn:hover,
QPushButton#CalendarLeftMonthBtn:pressed {
	image: url(:/img/images/previous_month_hover.png);
}
QPushButton#CalendarRightYearBtn {
	image: url(:/img/images/next_year.png);
}
QPushButton#CalendarRightYearBtn:hover,
QPushButton#CalendarRightYearBtn:pressed {
	image: url(:/img/images/next_year_hover.png);
}
QPushButton#CalendarRightMonthBtn {
	image: url(:/img/images/next_month.png);
}
QPushButton#CalendarRightMonthBtn:hover,
QPushButton#CalendarRightMonthBtn:pressed {
	image: url(:/img/images/next_month_hover.png);
}
QWidget#CalendarTopWidget {
	background-color: white;
	border-bottom: 1px solid gray;
}
QWidget#CalendarBottomWidget {
	background-color: white;
	border-top: 1px solid gray;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,3,0,1">
    <item>
     <widget class="QGroupBox" name="groupBoxTitle">
      <property name="title">
       <string>标题</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditTitle">
         <property name="placeholderText">
          <string>请输入标题（必填）</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxTime">
      <property name="title">
       <string>时间</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEditTime">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxClock">
         <property name="toolTip">
          <string>是否设置日程提醒</string>
         </property>
         <property name="icon">
          <iconset resource="sources.qrc">
           <normaloff>:/img/images/clock_light.png</normaloff>:/img/images/clock_light.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxAddress">
      <property name="title">
       <string>地点</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditAddress">
         <property name="placeholderText">
          <string>请输入地点（可选）</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxNote">
      <property name="title">
       <string>备注</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="widgetToolBar" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelFont">
            <property name="text">
             <string>字体：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFontComboBox" name="fontComboBox">
            <property name="currentText">
             <string>Adobe Arabic</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelFontSize">
            <property name="text">
             <string>大小：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxFontSize">
            <property name="value">
             <number>9</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditNote">
         <property name="placeholderText">
          <string>备注信息（可选）</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/img/images/logout.png</normaloff>:/img/images/logout.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>关闭新建日程</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/img/images/save.png</normaloff>:/img/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存日程</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>NewScheduleDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>actionQuit</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>actionSave</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
